index.bs

```diff
@@ -187,6 +186,108 @@ If the console is not open when the printer operation is called,
 implementations should buffer messages to show them in the future up to an
 implementation-chosen limit (typically on the order of at least 100).
 
+<h4 id="loglevel-severity">Indicating <var>logLevel</var> Severity</h4>
+
+This section groups together {{console}} methods whose <var>logLevel</var>s produce output
+indicating similar levels of severity.
+
+<div class="example" id="loglevel-example">
+  Since <code>log</code> and <code>debug</code> produce similarly formatted outputs, they are
+  grouped together by their log level severity in the following table.
+</div>
+
+<table>
+  <thead>
+    <tr>
+      <th>{{console}} method</th>
+      <th>Log Level Severity</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>log, trace, dir, dirxml, group, groupCollapsed, debug</code></td>
+    <td>log</td>
+    <td>
+      Console message with implementation-specific formatting designed to indicate a generic log
+    </td>
+  </tr>
+  <tr>
+    <td><code>count, info, timeEnd</code></td>
+    <td>info</td>
+    <td>
+      Console message with implementation-specific formatting designed to make the message stand
+      out as informative
+    </td>
+  </tr>
+  <tr>
+    <td><code>warn</code></td>
+    <td>warn</td>
+    <td>
+      Console message with implementation-specific formatting designed to warn the user of
+      something indicated by the message
+    </td>
+  </tr>
+  <tr>
+    <td><code>error, assert</code></td>
+    <td>error</td>
+    <td>
+      Console message with implementation-specific formatting designed to indicate an error to the
+      user
+    </td>
+  </tr>
+</table>
+
+While the above table groups together {{console}} methods by <var>logLevel</var> severity, the
+Printer is still implementation-defined, and is therefore allowed to provide formatting variations
+on a per-method basis while remaining compliant. Some common examples of this UX improvisation are
+as follows:
+
+<ul>
+  <li>
+    <code>timeEnd</code> custom formatting.
+    <div class="example" id="timeEnd-formatting">
+      Formatting applied to output produced by calls to <code>timeEnd</code> may be unique to this
+      method.
+
+      <img src="images/timeEnd-formatting.png" />
+    </div>
+  </li>
+  <li>
+    <code>count</code> does not always print new output.
+    <p class="XXX">TODO: A picture of this in the Edge console would be nice.</p>
+  </li>
+</ul>
+
+<h4 id="printer-ux-innovation">Printer UX Innovation</h4>
```

Comment:
Again no titlecase.

"UX" is an uncommon enough acronym in specs that it should probably be spelled out. It also might be clearer to just use the more common "UI" or "UI widgets" for the bullet points.